scripts/hab_conf_transfer_to_Q.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Transfer `habitat_confirmations.xls` data to `form_fiss_2023.gpkg`"
+date: "Created: 2024-04-26 | Updated: `r format(Sys.Date(), '%Y-%m-%d')`"
+output: 
+  html_document:
+    code_folding: "none"
+params:
+  repo_owner: "NewGraphEnvironment"
```

Comment:
i actually like these `repo_*` params in the yaml even if not used since they are ready to go for git, links etc.

Reply:
okay. I was copying from https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/blob/main/scripts/01_prep_inputs/0120-photos.Rmd#L7C1-L9C50 but can move them to here instead, https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/blob/main/index.Rmd#L27C1-L29C89